javascript/builtins/Array.json

```diff
@@ -1758,6 +1758,42 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "stable": {
+            "__compat": {
+              "description": "Stable",
+              "support": {
+                "chrome": {
+                  "version_added": "70"
+                },
```

Comment:
Please add values for chrome_android and webview_android.

Reply:
You got it.